proto/wippersnapper/uart/v1/uart.proto

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2023 Brent Rubell for Adafruit Industries
+// SPDX-License-Identifier: MIT
+syntax = "proto3";
+
+package wippersnapper.uart.v1;
+import "nanopb/nanopb.proto";
+import "wippersnapper/i2c/v1/i2c.proto";
+
+/**
+* UARTBusData represents a message to configure a UART bus for communication with a device.
+* NOTE: This message is never sent directly, it is packed inside UARTDeviceAttachRequest.
+*/
+message UARTBusData {
+  int32 baudrate        = 1; /** The baudrate to use for UART communication (may be a common baud rate such as: 1200bps, 2400bps, 4800bps, 19200bps, 38400bps, 57600bps, or 115200bps). */
+  string pin_rx         = 2[(nanopb).max_size = 6]; /** The pin on which to receive UART stream data. */
+  string pin_tx         = 3[(nanopb).max_size = 6]; /** The pin on which to transmit UART stream data. */
+  bool is_invert        = 4; /** Inverts the UART signal on RX and TX pins. Defaults to False. */
+}
+
+/**
+* UARTDeviceAttachRequest represents a message sent from IO to a device
+* to configure the UART bus (if not already configured) and attach a device.
+*/
+message UARTDeviceAttachRequest {
+  UARTBusData bus_info     = 1; /** The UART bus configuration. */
+  string device_id         = 2[(nanopb).max_size = 15]; /** The unique identifier of the device to attach to the UART bus, from Adafruit_WipperSnapper_Components. */
+  int32 polling_interval   = 3; /** The polling interval, in milliseconds, to use for the device. */
+}
+
+/**
+* UARTDeviceAttachResponse represents a message sent from a device to IO to
+* confirm that a device has been attached to the UART bus.
+*/
+message UARTDeviceAttachResponse {
+  string device_id = 1[(nanopb).max_size = 15]; /** The unique identifier of the device to attach to the UART bus, from Adafruit_WipperSnapper_Components. */
+  bool is_success  = 2; /** True if the UARTInitRequest was successful, False otherwise. */
+}
+
+/**
+* UARTDeviceEvent represents incoming data from a UART sensor.
+*
+message UARTDeviceEvent {
+  string device_id = 1[(nanopb).max_size = 15];                    /** The unique identifier of the device to attach to the UART bus, from Adafruit_WipperSnapper_Components. */
+  repeated wippersnapper.i2c.v1.SensorEvent sensor_event  = 2[(nanopb).max_count = 15]; /** A, optionally repeated, SensorEvent from a sensor. */
+}
```

Comment:
What's up here, is this commented out on purpose? If so, Line 44 accidentally breaks out of the doc-block due the line above closing itself with `*/`, and it's still being referenced in `signal.proto`.